client/java/src/main/java/com/linecorp/centraldogma/internal/client/ReplicationLagTolerantCentralDogma.java

```diff
@@ -681,6 +683,9 @@ private boolean handleRevisionNotFound(
             String projectName, String repositoryName, Revision revision, Throwable cause) {
 
         requireNonNull(cause, "cause");
+        if (cause instanceof CompletionException) {
```

Comment:
We can use `Exceptions.peel(cause)`

Reply:
Hm, we can use it only when under `client-java-armeria` or `server`.
This class is under `client-java` and it does not have armeria dependency.

Reply:
Ah yes, forgot about it. 😅 
So isn't there any chance that the exception is wrapped by other exceptions such as `ExecutionException`?